src/resources/elements/primeDesignSystem/demos/demos.ts

```diff
@@ -1,86 +1,19 @@
 import {PLATFORM} from "aurelia-pal";
-import {Router, RouterConfiguration} from "aurelia-router";
+import {Router} from "aurelia-router";
 
 export class Demos {
 
   router: Router;
 
-  private configureRouter(config: RouterConfiguration, router: Router) {
-    const routes = [
+  private configureRouter(config, router): void {
```

Comment:
Do we have to get rid of typing the arguments?

Reply:
Right, was lost from copy paste.